eth2near/eth2near-block-relay/eth-on-near-client.js

```diff
@@ -227,8 +239,46 @@ class EthOnNearClientContract extends BorshContract {
       process.exit(1)
     }
   }
+
+  async updateDagMerkleRoots (dagsStartEpoch) {
```

Comment:
Please add a check on `dagsStartEpoch` value that it's within the bounds of DAG roots.

Reply:
90d8133028de8ea8f7693953f78e44a01c20098e